pkg/util/mount/mount.go

```diff
@@ -76,3 +80,40 @@ func GetMountRefs(mounter Interface, mountPath string) ([]string, error) {
 	}
 	return refs, nil
 }
+
+// GetDeviceRefCount: given a device path, find its reference count from /proc/mounts
+// returns the reference count to the device and error code
+func GetDeviceRefCount(mounter Interface, deviceName string) (int, error) {
+	mps, err := mounter.List()
+	if err != nil {
+		return -1, err
+	}
+
+	// Find the number of references to the device.
+	refCount := 0
+	for i := range mps {
+		if strings.HasPrefix(mps[i].Device, deviceName) {
```

Comment:
HasPrefix makes me nervous because it could false-positive.  Why is this not == ?